ulm-semantics/main/encoding/encoder.md

```diff
@@ -12,7 +12,7 @@ module ULM-ENCODING-HELPER
     rule encodeHexBytes(_:String) => .Bytes
         [owise]
     rule encodeHexBytes(S:String)
-        => Int2Bytes(2, String2Base(substrString(S, 0, 2), 16), BE)
+        => Int2Bytes(1, String2Base(substrString(S, 0, 2), 16), BE)
```

Comment:
I understand the change in the signature size from 8 Bytes to 2, but what is the motivation for the change here?

Reply:
Two hex characters (`substrString(S, 0, 2)`) should become a single byte.

Reply:
i.e., "FA" should become b"\xfa" which is one byte